Framework/DataObjects/inc/MantidDataObjects/MDGridBox.tcc

```diff
@@ -1082,11 +1082,12 @@ TMDE(void MDGridBox)::centerpointBin(MDBin<MDE, nd> &bin,
  * @param radiusSquared :: radius^2 below which to integrate
  * @param signal [out] :: set to the integrated signal
  * @param errorSquared [out] :: set to the integrated squared error.
+ * @param innerRadiusSquared :: radius^2 above which to integrate
  */
 TMDE(void MDGridBox)::integrateSphere(API::CoordTransform &radiusTransform,
                                       const coord_t radiusSquared,
                                       signal_t &signal,
-                                      signal_t &errorSquared) const {
+                                      signal_t &errorSquared, const coord_t innerRadiusSquared) const {
```

Comment:
While it seems these .tcc files are not handled by clang-format, you could still consider avoiding long lines like this.